src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -112,24 +150,30 @@ public void stopAngledArm() {
 
     public void setMotor(double position, WPI_TalonFX motor) {
         if (state == ClimbSubsystemState.ENABLED) {
-            motor.set(ControlMode.Position, position);
+            System.out.println("motor set: " + position);
+            //motor.set(ControlMode.Position, position)   >:( bad
+            motor.set(ControlMode.PercentOutput, 1);
         }
     }
 
     public boolean isFixedFullyExtended() {
-        return climbFixedMotor.getSelectedSensorPosition() >= ClimbConstants.MAX_ENCODER_TICKS;
+        System.out.println(climbFixedMotor.getSelectedSensorPosition());
```

Comment:
It may be helpful to add a label before printing each sensor position - this way we'll know which arm's sensor position we're looking at, instead of only having a number. Displaying the positions on shuffleboard would be another good alternative.